src/modules/Crop.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Image Cropping module
```

Comment:
Can you paste in the usage and defaults to the comment area here, until we figure out where to more appropriately store that info? Just the content from the body of your original PR. Thanks!